model/websocket_client_test.go

```diff
@@ -63,8 +74,13 @@ func TestWebSocketClose(t *testing.T) {
 	// Therefore, it is still racy and can panic. There is no use chasing this
 	// again because it will be completely overhauled in v6.
 	t.Skip("Skipping the test. Will be changed in v6.")
```

Comment:
Are we still going to skip the rest?

Reply:
Yeah for now. We never got to it in v6. It's out of scope for this PR.